startables/test/test_startables.py

```diff
@@ -35,6 +35,12 @@ def some_df(self):
     def col_specs(self):
         return {n: ColumnMetadata(Unit(u)) for n, u in zip(['a', 'b', 'c'], ['-', 'text', 'm'])}
 
+    @fixture
+    def col_specs_with_format(self):
+        return {'a': ColumnMetadata(Unit('-'), format_str='${:,.2f}'),
+                'b': ColumnMetadata(Unit('text')),
+                'c': ColumnMetadata(Unit('m'), format_str='{0:1.4e}')}
```

Comment:
I guess the `0` in `{0:1.4e}' has no real effect? i.e. having anything else than `''` or `'0'` in its place would lead to an error?